### PR TITLE
improve(core): rename umip 15 scripts to make them general-purpose

### LINKED ISSUE #2439: Write and test deployment scripts to propose DVM upgrade (voting.sol) with ancillary data fields
Created by Matt Rice via monday.com integration. 🎉

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

UMIP-15 scripts can be reused for upgrading the Voting contract, generally. To signify this, comments and naming were changed throughout. Otherwise, the scripts seem to work as expected with the new contract.

**Summary**

Almost entirely naming/cosmetic changes to signify that these scripts and Solidity code do not just apply to UMIP-15.

The only minor change that I made was to symlink the Vote simulation script to the one that the other umip folders use.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

Note: I ran the code on a mainnet fork to verify that it ran correctly.

**Issue(s)**

Fixes #2439

### CODE REVIEW

File: packages/core/scripts/voting-upgrade-umip/1_Propose.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const Registry = artifacts.require("Registry");
 const Voting = artifacts.require("Voting");
 const VotingToken = artifacts.require("VotingToken");
 const Governor = artifacts.require("Governor");
-const Umip15Upgrader = artifacts.require("Umip15Upgrader");
+const VotingUpgrader = artifacts.require("VotingUpgrader");
```
Comment (chrismaree) [inline packages/core/scripts/voting-upgrade-umip/1_Propose.js:14]: here we are using `artifacts.require`. however, in PR https://github.com/UMAprotocol/protocol/pull/2444/files we are using `getTruffleContract`. should we consider changing this to use the same syntax everywhere?

**Review by chrismaree — APPROVED** [review]
Looks good! I had a nit on how we are importing things but otherwise, I think this is good to go

**Review by nicholaspai — APPROVED** [review]
LGTM

File: packages/core/scripts/voting-upgrade-umip/1_Propose.js (hunk 1 above)
Comment (mrice32) [reply packages/core/scripts/voting-upgrade-umip/1_Propose.js]: Good point. Fixed.

**Pull request merged**